apps/android/app/src/main/java/com/axon/app/ui/settings/SettingsScreen.kt

```
@@ -29,143 +49,310 @@ import tv.tootie.aurora.components.AuroraStatusIndicator
 import tv.tootie.aurora.components.AuroraStatusTone
 import tv.tootie.aurora.components.AuroraTextField
 
+private enum class SettingsTab(val label: String, val icon: ImageVector) {
+    Connection("Connection", Icons.Rounded.Link),
+    Env("Env", Icons.Rounded.Key),
+    Config("config.toml", Icons.Rounded.Slideshow),
+}
+
 @Composable
 fun SettingsScreen(vm: SettingsViewModel = viewModel()) {
```

detekt (1.23.8)

[warning] The function SettingsScreen appears to be too complex based on Cyclomatic Complexity (complexity: 26). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)